src/plugins/gui/amountQuotePlugin.ts

```diff
@@ -234,21 +244,27 @@ export const amountQuoteFiatPlugin: FiatPluginFactory = async (params: FiatPlugi
 
       logEvent(isBuy ? 'Buy_Quote' : 'Sell_Quote')
 
-      // Pick a default fiat amount that is roughly equal to DEFAULT_FIAT_AMOUNT
+      // Pick a default fiat amount in the foreign fiat amount that is roughly
+      // equal to the default USD fiat amount
       let initialValue1: string | undefined
-      if (displayFiatCurrencyCode !== 'USD' && defaultFiatAmount == null) {
+      if (displayFiatCurrencyCode !== 'USD' && pluginDefaultFiatAmount == null) {
```

Comment:
This change could be greatly simplified by declaring 

```
const startingFiatAmount = isLightAccount ? DEFAULT_FIAT_AMOUNT_LIGHT_ACCOUNT : (defaultFiatAmount ?? DEFAULT_FIAT_AMOUNT)
```

Then replace all use of `defaultFiatAmount` or `DEFAULT_FIAT_AMOUNT` with `startingFiatAmount`